docs/source/fundamentals.md

```diff
@@ -82,11 +84,7 @@ Based on how tools communicate with other tools, we can define 3 levels of opera
 
```

Comment:
We may want to think whether we want to associate Esperanto with SimPhoNy...

Reply:
Please check the latest version for an updated example